server/util/networking/networking.go

```diff
@@ -154,6 +154,15 @@ func DeleteRoute(ctx context.Context, vmIdx int) error {
 	return runCommand(ctx, "ip", "route", "delete", cloneIP)
 }
 
+func DeleteRuleIfSecondaryNetworkEnabled(ctx context.Context, vmIdx int) error {
+	if !IsSecondaryNetworkEnabled() {
+		// IP rule is only added when the secondary network is enabled
+		return nil
+	}
+	cloneIP := fmt.Sprintf("192.168.%d.%d", vmIdx/30, ((vmIdx%30)*8)+3)
```

Comment:
can you make a "getCloneIP(vmIdx int)" func since we have this in a couple places?

Reply:
Done